File: Assets/Scripts/CSharp/ScalesHandler.cs
using UnityEngine;
using System.Collections;

public class ScalesHandler : MonoBehaviour 
{	
	public float groupWidth;
	public float groupHeight;
	public float icoSize;
	
	void Start () 
	{
		icoSize = Screen.height * 0.2f;
		groupWidth = ((icoSize)*5f+4f*(Screen.height*0.01f));
		groupHeight = ((icoSize)+Screen.height*0.01f);
		transform.position = Camera.main.ScreenPointToRay(new Vector3(Screen.width / 2f - groupWidth/2f, Screen.height - groupHeight,0f)).GetPoint(5);
	}
	
	void Update () 
	{
		icoSize = Screen.height * 0.2f;
		groupWidth = ((icoSize)*5f+4f*(Screen.height*0.01f));
		groupHeight = ((icoSize)+Screen.height*0.01f);
		transform.position = Camera.main.ScreenPointToRay(new Vector3(Screen.width / 2f - groupWidth/2f, Screen.height - groupHeight,0f)).GetPoint(5);
	}
}

File: Assets/Scripts/ScalesHandler.js

var groupWidth:float;
var groupHeight:float;
var icoSize:float;
function Start () {
icoSize = Screen.height * 0.2;
groupWidth = ((icoSize)*5+4*(Screen.height*0.01));
groupHeight = ((icoSize)+Screen.height*0.01);
transform.position = Camera.main.ScreenPointToRay(Vector3(Screen.width / 2 - groupWidth/2, Screen.height - groupHeight,0)).GetPoint(5);
}

function Update () {
icoSize = Screen.height * 0.2;
groupWidth = ((icoSize)*5+4*(Screen.height*0.01));
groupHeight = ((icoSize)+Screen.height*0.01);
transform.position = Camera.main.ScreenPointToRay(Vector3(Screen.width / 2 - groupWidth/2, Screen.height - groupHeight,0)).GetPoint(5);
}